src/components/List.js

```diff
@@ -44,3 +47,16 @@ export default function List() {
     </div>
   );
 }
+
+const getItemsKeys = (shoppingList, searchTerm) => {
```

Comment:
I like how you contained the search logic in a separate function and using `removePunctuation` before searching is def a good idea Well Done 👍
Maybe later we can also highlight the `searchTerm` in the items while searching [like here](https://bitsofco.de/a-one-line-solution-to-highlighting-search-matches) will be a fancy addition to the app 💅

Reply:
Yes, I think highlighting the search term is a great idea. 👍 
Thank you so much for this good review, Sarah. ❤️